cmd/miner_client/src/stratum_client_service.rs

```diff
@@ -231,18 +231,18 @@ impl Inner {
                     request_id+=1;
                     match req {
                         Request::LoginRequest(login_req, s)=>{
-                            let req_str = build_request_string("login", &login_req, request_id).expect("build stratum login request failed never happen");
-                            debug!("stratum client send request:{}",req_str);
-                            if let Err(err) = self.sink.send(req_str).await{
+                            let req = build_request_string("login", &login_req, request_id).expect("build stratum login request failed never happen");
```

Comment:
this place use req_str because req is another variable, see #L232

Reply:
It has been changed to `message`.